client/packages/common/src/intl/locales/en/common.json

```diff
@@ -315,6 +317,7 @@
   "error.no-asset-log-reasons": "There are no asset log reasons to display.",
   "error.no-asset-view-permission": "You do not have permission to view assets.",
   "error.no-assets": "There are no assets to display.",
+  "error.no-campaigns": "No campaigns to display",
```

Comment:
Nit: change this to "There are no campaigns to display" for consistency with similar error messages